nss-day-cohort-48/rare-server-river-rats
# Nwm-updated-Login

#### Changes Made
1. Added file categories/request.py tags/request.py directory.
1. Modified file categories/__init__.py  models/__init__.py modified:   models/login.py,  rare_users/request.py, request_handler.py tags/__init__.pyand functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout nwm-first-branch
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions: check that a user is able to attempt a login, and register if not a user
    > Example: When user attempts to login without a password, it should prompt the user to register. Should happen.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
